src/components/Events/EventCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Row, Col } from '../grid'
+import StyledLink from '../Common/StyledLink'
+
+import { CardTitle, BodyPrimary, CalendarDay } from '../Typography'
+import remcalc from 'remcalc'
+
+const CalendarPage = styled.div`
+  padding: 24px;
+  border: solid;
+  border-color: ${props => props.theme.colors.border};
+  border-width: thin;
+  width: 108px;
+  height: 108px;
+  text-align: center;
+
+  ${breakpoint('tablet')`
+    width: 143px;
+    height: 143px;
+  `}
+`
+
+const CenteredBodyPrimary = styled(BodyPrimary)`
+  text-align: center;
+  line-height: ${remcalc(18)};
+
+  ${breakpoint('tablet')`
+    line-height: ${remcalc(24)};
+
+  `}
+`
+
+const NoPaddingCol = styled(Col)`
+  padding: 0;
+`
+
+const DateCard = ({ date }) => (
+  <CalendarPage>
+    <CalendarDay noPadding>{date.day}</CalendarDay>
+    <CenteredBodyPrimary noPadding>
+      {date.month.toUpperCase()}
+    </CenteredBodyPrimary>
+  </CalendarPage>
+)
+
+const AlignRightCol = styled(Col)`
+  ${breakpoint('tablet')`
+text-align: right;
+
+  `}
+`
+
+const SelectivePaddingCardTitle = styled(CardTitle)`
+  padding-top: 0;
+  padding-bottom: ${remcalc(6)};
+
+  ${breakpoint('tablet')`
+padding-top: 0;
+
+  `}
+`
+
+const SelectivePaddingBodyPrimary = styled(BodyPrimary)`
+  padding-top: ${remcalc(24)};
+  padding-bottom: 0;
+
+  ${breakpoint('tablet')`
+padding-top: 0;
+
+  `};
+`
+
+const MeetupDetails = styled.div`
+  padding-top: ${remcalc(12)};
+  padding-bottom: ${remcalc(12)};
+`
+
+const EventCard = ({ event }) => {
+  const {
+    type,
+    date,
+    eventName,
+    eventLocation,
+    startTime,
+    endTime,
+    attendees = 0,
+    link
+  } = event
+
+  return (
+    <Row>
+      <NoPaddingCol width={[1, 1, 1, 1, 3 / 12, 2 / 12, 2 / 12]}>
+        <DateCard date={date} />
+      </NoPaddingCol>
+
+      <Col width={[1, 1, 1, 1, 6 / 12, 6 / 12, 5 / 12]}>
+        <SelectivePaddingBodyPrimary>{type}</SelectivePaddingBodyPrimary>
+        <MeetupDetails>
```

Comment:
Above in `<SelectivePaddingBodyPrimary>` you're remove the padding bottom but then assigning the same padding-top (12px) to  `<MeetupDetails>`. 

I don't think you need the wrapper here, you can allow the typography to set the spacing alone. 

I can tell from the designs that there are changes to the standard typography we have.

A suggestion:
Remove allll of the custom styled components here, just use the common typography components we have already have, see how that looks and then go from there.

Reply:
So MeetupType (eg Meetup / Conf / Workshop), needs to have padding-top 24 in mobile view (0 otherwise), and on all other views it needs to sit flush with the top of the EventCard component. The easiest option I can see for that is to assign a modifier noPadding to text and give it padding as needed via a div.

Thinking about this - if you're being given a spec like we have, how do you figure out which element gets to be assigned the padding, or is it "what is easier at the time"?

Reply:
You have two ways of doing this, either set the margin top on the typography component or set margin-bottom on the Date component. I would opt for the Date component to have the margin bottom mainly due to the fact that we have control of it and its a new component. As mentioned elsewhere, changing paddings on Common components is a last resort.

The general convention I follow is that spacing such as margin and padding should flow downwards, meaning we opt for [margin/padding]-bottom. If you start throwing spacing on some top and and others bottom and include it in different viewports in media queries you start to add complexity and make a bit of mess, having things on the bottom makes it easier to understand and work with. 

Importantly, this rule doesn't cover every single scenario, if its not possible to use bottom spacing then top is fine.